tests/integrationv2/utils.py

```diff
@@ -54,8 +54,6 @@ def invalid_test_parameters(*args, **kwargs):
             ] if provider_
     ]
     # autopep8: on
-    # Always consider S2N
-    providers.append(S2N)
```

Comment:
Just curious - how did this impact the criterion stuff? I would have expected the invalid test parameters to be the same between S2N and CriterionS2N.

Reply:
Agreed. This may be a symptom of a more serious problem. Maybe we're not building either CriterionS2N or the regular S2N with the right libcrypto? What's different, which test cases are failing?

Reply:
Without the import, it was failing across the board because of xdist, which is mentioned https://github.com/aws/s2n-tls/blob/main/tests/integrationv2/README.md#troubleshooting